examples/jsm/node-editor/NodeEditor.js

```diff
@@ -161,12 +164,14 @@ export const NodeList = [
 				name: 'Trigonometry',
 				icon: 'wave-sine',
 				tip: 'Sin / Cos / Tan / ...',
+				tags: 'sin, cos, tan, asin, acos, atan',
```

Comment:
I think tags can be more expanded as `sine`, `cosine`, `arcsine`, etc? So that they can be found both by queries like `asin` and `arcsine`.

Reply:
Sounds good, but I think it's work for other PR...